### ISSUE TITLE
Unable to reproduce the results

### ISSUE THREAD

**Issue by GodXuxilie** [LijieFan/AdvCL#6]
Dear Authors,

Thanks for your interesting paper and the code! I encountered the issue when I reproduced your results.

I conducted AdvCL on CIFAR-10 by using your pre-training code and your finetuning code to conduct SLF on CIFAR-10. I got the following results copied from the generated log of your finetuning code:
```
best accuracy: 10.43
best accuracy clean: 13.14
```
In addition, I repeated the experiments including the pre-training and the finetuning procedure again.  The following results are copied from the generated log:
```
best accuracy: 10.92
best accuracy clean: 14.12
```

Compared with your reported results on CIFAR-10 (standard test accuracy is 80.85% and PGD-20 robust test accuracy is 50.45%), my reproduced results are **significantly lower** than what you reported. 

Could you help me figure out the reasons and guide me to reproduce your results?  Thank you!
